components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/RefreshGrantHandler.java

```diff
@@ -574,4 +574,20 @@ private static void addUserAttributesToCache(AccessTokenDO accessTokenBean,
             AuthorizationGrantCache.getInstance().addToCacheByToken(authorizationGrantCacheKey, grantCacheEntry);
         }
     }
+
+    private boolean getRenewRefreshTokenProperty(String renewRefreshToken) {
+
+        if (renewRefreshToken != null) {
```

Comment:
shall we use StringUtils.isNotBlank here?

Reply:
fixed with 6c8dd0427e303578e81605a727761d4d05981af5